src/test/java/games/strategy/ui/SwingWorkerCompletionWaiterTest.java

```diff
@@ -1,24 +1,20 @@
 package games.strategy.ui;
 
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 
 import java.beans.PropertyChangeEvent;
 
 import javax.swing.SwingWorker;
 
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(MockitoJUnitRunner.class)
 public final class SwingWorkerCompletionWaiterTest {
```

Comment:
Java 9 is not the problem, I was experiencing this issue for a long time and simply ignored it.
The problem seems to have something to do with the `@InjectMock` Annotation.
I will post the stacktrace once I'm back at home.